Enforcer plugin configuration wrongly defines bannedRepository

I experience build failure with error
```quote
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java-version) on project wsat-simple: Unable to parse configuration of mojo org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce for parameter bannedRepositories: Cannot assign configuration entry 'bannedRepositories' with value 'http://*' of type java.lang.String to property of type java.util.List -> [Help 1]
```

The reason seems to be the `bannedRepositories` are configured in wrong way. Based on the documentation at least https://maven.apache.org/enforcer/enforcer-rules/bannedRepositories.html

--- a/pom.xml
+++ b/pom.xml
@@ -698,8 +698,12 @@
                 <rules>
                   <bannedRepositories>
                       <level>${insecure.repositories}</level>
-                      <bannedRepositories>http://*</bannedRepositories>
-                      <bannedPluginRepositories>http://*</bannedPluginRepositories>
+                      <bannedRepositories>
+                          <bannedRepository>http://*</bannedRepository>
+                      </bannedRepositories>
+                      <bannedPluginRepositories>
+                          <bannedPluginRepository>http://*</bannedPluginRepository>
+                      </bannedPluginRepositories>
                   </bannedRepositories>
                   <requireJavaVersion>
                     <message>To build this project JDK ${jdk.min.version} (or greater) is required. Please install it.</message>

